error_class.js

```diff
@@ -0,0 +1,61 @@
+Error.prototype.createInnerHtml = function() {
+  return '天気情報が取得できませんでした。<br>時間をおいてから再度お試しください。';
+}
+
+class ClientError extends Error {
+  constructor(message) {
+    super(message);
+    this.name = this.constructor.name;
+  }
```

Comment:
呼び名って定数なので、staticってやつで定義するのがええんやないかな
そして定数はconstructorで定義するものではないです
```suggestion
class ClientError extends Error {
  static ErrorName = 'Client Error'

  constructor(message) {
    super(message);
  }
```

Reply:
https://github.com/KenJamesHioki/weather_app/pull/16/commits/a9b68bd0bd89393b07403decfaf7f6a7d7f1ba1c
エラーの名前を表示することは不必要だと感じたため、クラス内でnameプロパティを扱うことをやめました。
エラーメッセージとstatusコードのみをコンソールに表示する実装に変更しました。